build/lib/src/builder/post_process_build_step.dart

```diff
@@ -9,25 +9,30 @@ import 'package:async/async.dart';
 import 'package:build/build.dart';
 import 'package:crypto/src/digest.dart';
 
-import '../asset_graph/graph.dart';
-import '../asset_graph/node.dart';
+// This is not exported to hack around a package-private constructor.
+PostProcessBuildStep postProcessBuildStep(
+        AssetId inputId,
+        AssetReader reader,
+        AssetWriter writer,
+        void Function(AssetId) addAsset,
```

Comment:
nit: please add some comments for when/how these should be used

Reply:
Will add them at `runPostProcessBuilder` since that is where these are exposed.